In [1]:
import pandas as pd
df1 = pd.read_csv('customers.csv')

In [2]:
df1

,Channel,Region,Fresh,Milk,Grocery,Frozen,Detergents_Paper,Delicatessen
0,2,3,12669,9656,7561,214,2674,1338
1,2,3,7057,9810,9568,1762,3293,1776
2,2,3,6353,8808,7684,2405,3516,7844
3,1,3,13265,1196,4221,6404,507,1788
4,2,3,22615,5410,7198,3915,1777,5185
...,...,...,...,...,...,...,...,...
435,1,3,29703,12051,16027,13135,182,2204
436,1,3,39228,1431,764,4510,93,2346
437,2,3,14531,15488,30243,437,14841,1867
438,1,3,10290,1981,2232,1038,168,2125


In [3]:
df1.shape

(440, 8)

In [4]:
df1.info()

<class 'pandas.DataFrame'>
RangeIndex: 440 entries, 0 to 439
Data columns (total 8 columns):
 #   Column            Non-Null Count  Dtype
---  ------            --------------  -----
 0   Channel           440 non-null    int64
 1   Region            440 non-null    int64
 2   Fresh             440 non-null    int64
 3   Milk              440 non-null    int64
 4   Grocery           440 non-null    int64
 5   Frozen            440 non-null    int64
 6   Detergents_Paper  440 non-null    int64
 7   Delicatessen      440 non-null    int64
dtypes: int64(8)
memory usage: 27.6 KB


In [5]:
df1.describe().round(2)

,Channel,Region,Fresh,Milk,Grocery,Frozen,Detergents_Paper,Delicatessen
count,440.00,440.00,440.00,440.00,440.00,440.00,440.00,440.00
mean,1.32,2.54,12000.30,5796.27,7951.28,3071.93,2881.49,1524.87
std,0.47,0.77,12647.33,7380.38,9503.16,4854.67,4767.85,2820.11
min,1.00,1.00,3.00,55.00,3.00,25.00,3.00,3.00
25%,1.00,2.00,3127.75,1533.00,2153.00,742.25,256.75,408.25
50%,1.00,3.00,8504.00,3627.00,4755.50,1526.00,816.50,965.50
75%,2.00,3.00,16933.75,7190.25,10655.75,3554.25,3922.00,1820.25
max,2.00,3.00,112151.00,73498.00,92780.00,60869.00,40827.00,47943.00


In [8]:
df1["Channel"].unique()

array([2, 1])

In [7]:
df1["Region"].unique()

array([3, 1, 2])

In [9]:
df1["Channel"].value_counts()

Channel
1    298
2    142
Name: count, dtype: int64

In [10]:
df1["Region"].value_counts()

Region
3    316
1     77
2     47
Name: count, dtype: int64

Este dataset tiene 440 filas y 8 columnas
En este dataset todas las columnas son de tipo int64 y tampoco tiene valores unicos

La columna Channel tiene solo dos valores unicos: 1 y 2
La columna Region tiene solo tres valores unicos: 1, 2, 3

La media y la mediana nos indican que la mayoria de estan sesgados hacia valores altos

Tipo de variables: 
Numerica continua: Fresh, Milk, Grocery, Frozen, Detergents_Paper, Delicatessen
Numerica discreta: Channel, Region

